### ISSUE TITLE
New warnings: external not defined

### ISSUE THREAD

**Issue by jjceresa** [FluidSynth/fluidsynth#491]
Building using VS 2010.
From actual master version 2.0.2

At compile time of fluidsys.c, and fluidcmd.c we get these new externals not defined.
```
3>..\..\fluidsynth\src\utils\fluid_sys.c(1260): warning C4013: 'read' not defined ; extern int used by default
3>..\..\fluidsynth\src\utils\fluid_sys.c(1342): warning C4013: 'write' not defined ; extern int used by default

3>..\..\fluidsynth\src\bindings\fluid_cmd.c(581): warning C4013: '_open' not defined; extern int used by default
3>..\..\fluidsynth\src\bindings\fluid_cmd.c(595): warning C4013: '_close' not defined ; extern int used by default
```
Note that there is no problem at link time (i.e no unresolved symbols), and the dll is build successfully.
These are only new warnings. It seems that   `#include <io.h>` is missing somewhere. 

For information, v2.0.0 build successfully without warning and in fluidsynth_priv.h, io.h is included as this:
```
#if HAVE_IO_H
#include <io.h>
#endif
```

**Comment by derselbst** [comment]
Sry, I removed it because I thought it's unused and didn't pay enough attention to the warnings. Fixed and clarified in d2078e63b4656c4984972b8fbdbae4281f8550aa.

**Comment by jjceresa** [comment]
Thanks !. Now compile without warnings.
